src/main/resources/application.properties

```diff
@@ -0,0 +1,4 @@
+
+spring.datasource.url=jdbc:mysql://127.0.0.1:3306/myschema
+spring.datasource.username=root
+spring.datasource.password=abc123
```

Comment:
Good practice is not not store the username & password. It should be passed in a env variable.